Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
+  override GOGCFLAGS += -N -l
```

Comment:
Cool.  Learned a new trick today. So override doesn't really override but not overriding does. That "makes" sense ;-)